moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -118,6 +128,14 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     return l;
   }
 
+  std::string getJointValueTarget()
```

Comment:
Same here. Why do you want to expose the internal RobotState?

Reply:
It's just a copy of the state, not quite exposing the internal rs. This compliments above; yes this might be possible to instead use a list like get Joint Value Target with joint states but does not have a clear explicit way to handle multi dof joints